includes/admin/views/addons/addon-item.php

```diff
@@ -17,9 +17,14 @@
 		<a class="llms-add-on-link" href="<?php echo esc_url( $addon->get_permalink() ); ?>" target="_blank">
 
 			<header>
-				<object data="<?php echo esc_url( llms()->plugin_url() . '/assets/images/addons/' . basename( $addon->get( 'image' ) ) ); ?>">
-					<img alt="<?php echo $addon->get( 'title' ); ?> Banner" src="<?php echo esc_url( $addon->get( 'image' ) ); ?>">
-				</object>
+				<?php
+				if ( is_readable( llms()->plugin_path() . '/assets/images/addons/' . basename( $addon->get( 'image' ) ) ) ) {
```

Comment:
first check if  `basename( $addon->get( 'image' ) ) ` is not empty, otherwise:
`if ( is_readable( llms()->plugin_path() . '/assets/images/addons/' . basename( $addon->get( 'image' ) ) ) ) {` will always return true, being the directory readable. right?

Reply:
done 👍